src/Storage/Commands.Storage/Blob/Cmdlet/StartAzureStorageBlobCopy.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     using System.Security.Permissions;
     using System.Threading.Tasks;
 
-    [Cmdlet(VerbsLifecycle.Start, StorageNouns.CopyBlob, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = ContainerNameParameterSet),
+    [Cmdlet(VerbsLifecycle.Start, StorageNouns.CopyBlob, SupportsShouldProcess = true, DefaultParameterSetName = ContainerNameParameterSet),
```

Comment:
What's the default ConfirmImpact?  We would like to have the ConfirmImpact= low for this cmdlet.